tests/integration/tier0/rollback-handling/test_rollback_handling.py

```diff
@@ -165,8 +186,8 @@ def test_terminate_on_username_prompt(convert2rhel):
     Verify that c2r goes successfully through the rollback.
     """
     with convert2rhel("--debug -y --no-rpm-va") as c2r:
-        c2r.expect("Username:")
-        terminate_and_assert_good_rollback(c2r)
+        if c2r.expect("Username:") == 0:
```

Comment:
I would add `timeout` here as well. Since it is everywhere above. Or is there a reason not to have timeout here?